<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1413</width>
    <height>914</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="connectBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>730</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disBtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>730</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QLabel" name="SerialPlot">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial plot</string>
    </property>
   </widget>
   <widget class="QLabel" name="SerialPrint">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial print</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotData" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>110</y>
      <width>651</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>650</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ComPort">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>650</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>COM port</string>
    </property>
   </widget>
   <widget class="QComboBox" name="baudBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>680</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="baudRate">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>680</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Baudrate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputLine">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>551</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>120</y>
      <width>651</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="exportBtn">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>80</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileLine">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>80</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="runBtn">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>660</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseBtn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>660</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QComboBox" name="extenCombo">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>.csv</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.xlxs</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.txt</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pauseBtn_2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>660</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseBtn_3">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>660</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>730</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>650</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
